### DISCUSSION TITLE
Help you?

### DISCUSSION THREAD

**Discussion by WapackDev** [mpdifran/Memento-Issues#68] [Ideas]
Hi Mark,

I'm just beginning to learn how to program (as a semi-retirement gig) so I can't help on that front, but I like what you are doing with Memento, and would like to know if there is anything I can do to help, like with "Help", or some basic documentation, like a user manual.

I think there is huge potential for an app like Memento, specifically providing a better way to interact with Reminders, but it seems like you are working on this alone and on the side?? 

WD66

**Comment by mpdifran** [comment]
Sure, I'd love some help with FAQs / documentation etc!
